src/ui/public/agg_table/agg_table.html

```diff
@@ -3,6 +3,7 @@
   rows="rows"
   columns="formattedColumns"
   per-page="perPage"
+  link-to-top="true"
```

Comment:
Let's remove it from here, since it looks kind of weird in the context of a Dashboard, and also doesn't function correctly:

![image](https://cloud.githubusercontent.com/assets/1238659/22448218/aa566d42-e70c-11e6-9be3-e853eade48f4.png)